src/common/legacy_config_opts.h

```diff
@@ -844,26 +843,6 @@ OPTION(kinetic_user_id, OPT_INT) // kinetic user to authenticate as
 OPTION(kinetic_hmac_key, OPT_STR) // kinetic key to authenticate with
 OPTION(kinetic_use_ssl, OPT_BOOL) // whether to secure kinetic traffic with TLS
 
-
-OPTION(rocksdb_separate_wal_dir, OPT_BOOL) // use $path.wal for wal
-SAFE_OPTION(rocksdb_db_paths, OPT_STR)   // path,size( path,size)*
-OPTION(rocksdb_log_to_ceph_log, OPT_BOOL)  // log to ceph log
```

Comment:
ditto, please add `.set_description()` for the corresponding option in `options.cc`.